docs/index.html

```diff
@@ -2,13 +2,13 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
-		<title>three.js docs</title>
+		<title>three.js docs/examples</title>
```

Comment:
I wouldn't change the titles.

Reply:
> I never found the time to ask this in @CodyJasonBennett's PR...
> 
> Instead of the `make-docs` script... How crazy would it be to generate the list at load time?
> 
> Something along these lines:
> 
> ```
> <script type="module" defer>
> 
> import * as THREE from '../build/three.module.js';
> 
> const list = [];
> 
> function parseTHREE() {
> 
>   for ( const name in THREE ) {
>    
>     const property = THREE[ name ];
> 
>     if ( typeof property === 'function' ) {
> 
>       list.push( parseFunction( property );
> 
>     } else {
> 
>       list.push( property );
> 
>     }
> 
> }
> 
> function parseFunction( f ) {
> 
>     const list = [];
>     const object = new f();
> 
>     for ( const name in object ) {
> 
>         list.push( name );
> 
>     }
> 
>     return list;
> 
> }
> 
> parseTHREE();
> 
> </script>
> ```
> 
> Maybe in a worker?

That's not possible because it requires opening every  single HTML file. We certainly don't want to do that online.
And the generated file is only 69KB, and you only need to generate it sometimes. it's fine to keep it.

Reply:
Actually, it could be done at run time, if we load the THREE.js, which I'm now going to load again...

Reply:
> I wouldn't change the titles.

Okay, but I did it because now 1 page can show both the docs and the examples. When we switch tabs, the code doesn't load another page, for improved speed.
I'm going to revert it, and then dynamically change the title when we switch tabs.